rooms.js

```diff
@@ -1149,12 +1149,17 @@ class BasicChatRoom extends BasicRoom {
 		if (this.staffMessage && user.can('mute', null, this)) message += (message ? '<br />' : '\n|raw|<div class="infobox">') + '(Staff intro:)<br /><div>' + this.staffMessage.replace(/\n/g, '') + '</div>';
 		if (this.modchat) {
 			message += (message ? '<br />' : '\n|raw|<div class="infobox">') + '<div class="broadcast-red">' +
-				'Must be rank ' + this.modchat + ' or higher to talk right now.' +
+				`Must be rank ${this.modchat} or higher to talk right now.` +
+				'</div>';
+		}
+		if (this.modjoin) {
+			message += (message ? '<br />' : '\n|raw|<div class="infobox">') + '<div class="broadcast-red">' +
+				`Must be rank ${this.modjoin} or higher to join the room right now.` +
```

Comment:
I guess since roomvoices can join regardless of the modjoin setting this would probably be better as ``Must be room voice or equal to ${this.modjoin} to join the room right now``

Reply:
And maybe even through in a “Users”